modules/libcom/RTEMS/epicsMakeMemFs.pl

```diff
@@ -67,19 +130,124 @@
 };
 
 EOF
-  $N++;
-}
+        $N++;
+    }
 
-my $files = join ', ', map "&file_${_}", (0 .. $N-1);
+    my $files = join ', ', map "&file_${_}", (0 .. $N-1);
 
-print $DST <<EOF;
+    print $DST <<EOF;
 static const epicsMemFile* files[] = {
-  $files, NULL
+$files, NULL
 };
 
 static
 const epicsMemFS ${varname}_image = {&files[0]};
 const epicsMemFS * $varname = &${varname}_image;
 EOF
 
+
+} elsif($opt_f eq "imf") { # ================ Format 2
+
+    my $inputs = join "\n *    ", @ARGV;
+    print $DST <<EOF;
+/* $outfile containing
+ *    $inputs
+ */
+
+#include <epicsStdio.h>
+EOF
+    my $N = 0;
+    my (@fencode, @alens, @clens);
+
+    for my $fname (@ARGV) {
+        my $realfname = $fname;
+
+        $fname = File::Spec->canonpath($fname);
+        $fname = File::Spec->rel2abs($fname);
+        $fname = File::Spec->abs2rel($fname, $opt_T);
+
+        # attempt Windows to unix path convert
+        $fname =~ s(\\)(/)g;
+
+        # strip leading "../" "./" or "/"
+        $fname =~ s(^\.{0,2}/)()g;
```

Comment:
If you want to drop multiple leading `/`, `./` and/or `../` components, adjust the comment use this regexp:

```perl
$fname =~ s[^(\.{0,2}/)+][];
```